### PR TITLE
adds GHA info and link to docs to readme for playwright package

### PR DESCRIPTION
**Pull request by ceceliacreates**

### CODE REVIEW

File: packages/playwright/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Provides utilities to support using [Replay](https://replay.io) with [Playwright](https://playwright.dev)
 
+Use with [action/playwright](https://github.com/Replayio/action-playwright) to automatically upload replays of failed tests. [Check out our documentation here.](https://docs.replay.io/docs/recording-playwright-b62474b5aadd49e2b0c44a7580b0617e#4f9d9bb360974bf7942f8edae8dcd742)
```
Comment (ryanjduffy) [inline packages/playwright/README.md:5]: `action-playwright` instead of `action/playwright`

Comment (ryanjduffy) [reply packages/playwright/README.md:5]: or maybe `replayio/action-playwright` for ease of copying

Comment (ceceliacreates) [reply packages/playwright/README.md]: Fixed! Thank you.

**Comment by ryanjduffy** [comment]
Removed the `@` since this is a GitHub reference and not an NPM package reference. Otherwise, :shipit:

**Review by ryanjduffy — APPROVED** [review]

**Pull request merged**